conda_env/specs/yaml_file.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from .. import env
+
+
+class YamlFileSpec(object):
+    def __init__(self, filename=None, name=None, **kwargs):
+        self.filename = filename
+        self.name = name
+
+    def can_handle(self):
+        self._environment = env.from_file(self.filename)
```

Comment:
If `self.filename` does not exist we are going to have an exception here. The exception is being captured in `main_create` module and we are showing a file not exist error. That's fine, but `args.filename` is not mandatory and it shouldn't because we can create an environment from a remote location. Looks like YamlFileSpec should be the last spec to execute. If that's the case we will need to ensure `all_specs` order and that seems to be strange. Makes sense?